packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -92,7 +96,10 @@ export interface CheckoutEvents {
 	 * @param getRevertible - a function provided that allows users to get a revertible for the change. If not provided,
 	 * this change is not revertible.
 	 */
-	changed(data: CommitMetadata, getRevertible?: RevertibleFactory): void;
+	changed(
+		data: CommitMetadata,
+		getRevertible?: RevertibleFactory | RevertibleAlphaFactory,
```

Comment:
```suggestion
		getRevertible?: RevertibleAlphaFactory,
```

Right? Why would the changed event ever provide a factory that exclusively creates Revertibles without a `clone()` method?